praw/models/reddit/subreddit.py

```diff
@@ -1447,6 +1457,11 @@ def update(
             valid emoji string, for example ``':snoo:'``.
         :param max_emojis: (int) Maximum emojis in the flair
             (Reddit defaults this value to 10).
+        :param fetch: Whether or not PRAW will fetch existing information on
+            the existing flair before updating (Default: False).
```

Comment:
The default is now `True`, so that should be updated here.

Reply:
Thank you.